api/lib/domain/models/AssessmentResult.js

```diff
@@ -40,17 +42,19 @@ class AssessmentResult {
       emitter,
       commentForJury: error.message,
       pixScore: 0,
+      reproducibilityRate: 0,
       status: status.ERROR,
       assessmentId,
       juryId,
     });
   }
 
-  static buildStandardAssessmentResult({ pixScore, status, assessmentId, juryId, emitter }) {
+  static buildStandardAssessmentResult({ pixScore, reproducibilityRate, status, assessmentId, juryId, emitter }) {
     return new AssessmentResult({
       emitter,
       commentForJury: 'Computed',
       pixScore: pixScore,
+      reproducibilityRate: reproducibilityRate,
```

Comment:
Ca se simplifie, non ?
```suggestion
      reproducibilityRate,
```